### ISSUE TITLE
Immortal treasure II bug

### ISSUE THREAD

**Issue by Meizner** [ValveSoftware/Dota2-Gameplay#3103]
I've had this happen to me multiple times now. I unlock the treasure in my battle pass, and when I want to open it it shows as already been opened in my account I didn't receive any items I double checked my inventory. It nowhere to be found and this has happened multiple times. It is frustrating as I have spent money to get these cases specifically.

**Comment by Meizner** [comment]
Guess only certain people will get a form of a response.

**Comment by Randomd0t** [comment]
I have a similar issue with immortal treasure 2 chests. When I receive them they cannot be left-clicked and right-click only allows to add the item to "Tag", "Add to collection", or "Delete". As if it is some unusable item like the vials used to paint river ingame, etc.

**Comment by Meizner** [comment]
Yeah, it's exactly that.

**Comment by Randomd0t** [comment]
Not sure what to do about it... the odds of any developers seeing this is minimal. Yet the issue is quite big. Not receiving chests that are paid for through battle-pass is a big deal. We can't be the only ones this is happening to.

**Comment by Randomd0t** [comment]
Ooo... I just read the immortal 2 chests will be available/able to open when part 2 of the battlepass begins. So it is likely not a bug after all it seems

**Comment by Meizner** [comment]
Yes, good eye there. So it not a bug its actually a feature lol. Thank you for clearing it up, this was killing me.
